btr-web/btr-main-app/services/file-significant-individual.ts

```diff
@@ -152,8 +152,8 @@ const submitSignificantIndividualFiling = async (sif: SignificantIndividualFilin
   return { data: data.value, error: error.value }
 }
 
-const getBtrFiling = async (businessIdentifier: string) => {
-  const url = `${constructBtrApiURL()}/plots/entity/${businessIdentifier}`
+const getBtrFiling = async (businessIdentifier: string, account?: string) => {
+  const url = `${constructBtrApiURL()}/plots/entity/${businessIdentifier}${account ? `?account_id=${account}` : ''}`
```

Comment:
please remove these UI changes - the account id is already added into the header of the request

Reply:
Sorry, that was my mistake, I suggested him this. Is there a reason why we should use the one from the headers, rather then using the path and/or query params?

To me it looks like it could be kinda messy to have some data in headers, some data in path.

Though, I could understand that authorization data stay in header.

Reply:
we use the account id in all the apis from the header and we do it already in the btr-api. I see no reason to change that flow. It's also not part of the path, it's being added as an arg here